klippy/mathutil.py

```diff
@@ -135,3 +135,78 @@ def matrix_sub(m1, m2):
 
 def matrix_mul(m1, s):
     return [m1[0]*s, m1[1]*s, m1[2]*s]
+
+
+######################################################################
+# Simple statistics
+######################################################################
+
+# Standard deviation (square-root of variance) of the given samples
+def std(samples) :
+    mean = 0.
+    for v in samples :
```

Comment:
I think it would be preferable to put all this code in load_cell_probe.py for now (and not alter mathutil.py for now).  We can refactor the code at a later date if needed.  Linear regressions are used in several areas of the code today, and it's not clear to me that this implementation is a good match for all the use cases.  Many times we want to implement a "rolling average" linear regression, or at least calculate the linear regression as samples arrive (via accumulating sum and accumulating sum of squares).  For this PR, simpler to just keep all the code in load_cell_probe.py .